### PR TITLE
Add the VoterWrapperAdapter for the dummy scheme

### PR DESCRIPTION
**Pull request by beagleknight**
This applies the same pattern that we saw in #106. The idea is to extract the dummy voting scheme part from the voter and use an adapter. Then we will be able to do the same with `election_guard`.

### CODE REVIEW

**Review by leio10 — APPROVED** [review]
Willing to see next PR!! 😍

**Review by orlera — APPROVED** [review]
🚂

**Review by agustibr — APPROVED** [review]

File: voting-scheme-dummy/src/voter_wrapper_adapter.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   * @param {Object} params - An object that contains the initialization params.
+   * - {String} voterId - The unique id of a trustee.
+   */
+  constructor({ voterId }) {
+    this.voterId = voterId;
+    this.wrapper = new VoterWrapper({ voterId });
+  }
+
+  /**
+   * Process the message and update the wrapper status.
```
Comment (slickepinne) [inline voting-scheme-dummy/src/voter_wrapper_adapter.js:21]:
```suggestion
   * Processes the message and updates the wrapper status.
```

**Review by slickepinne — APPROVED** [review]
Amazing!

File: voting-scheme-dummy/src/voter_wrapper_adapter.js (hunk 1 above)
Comment (beagleknight) [reply voting-scheme-dummy/src/voter_wrapper_adapter.js:21]: Discussed on slack to apply them in #110

**Pull request merged**